src/ui/menu_nbgl.c

```diff
@@ -31,7 +31,8 @@ void app_quit(void) {
 }
 
 void ui_menu_main(void) {
-    nbgl_useCaseHome(APPNAME, &C_app_boilerplate_64px, NULL, false, ui_menu_about, app_quit);
+ #define SETTINGS_BUTTON_DISABLED (false)
```

Comment:
```suggestion
 #define DISPLAY_SETTINGS_BUTTON (false)
```
?

Reply:
not sure.
If I have 
`nbgl_useCaseHome(APPNAME, &C_app_boilerplate_64px, NULL, DISPLAY_SETTINGS_BUTTON, ui_menu_about, app_quit);`
I don't know if the settings is displayed or not, the `#define` value should not be considered (especially if this `#define` is included in a header file subsequently)